db-developments/sql/_procedures.sql

```diff
@@ -120,4 +123,4 @@ BEGIN
         RAISE NOTICE '( % ) is not a valid field for function apply_correction to ( % )', _field, _table;    
```

Comment:
Let's make this `RAISE EXCEPTION` so that if we're calling this with bad values that the call fails?